phase1/gce/configure-vm-kubeadm.sh

```diff
@@ -27,7 +27,7 @@ fi
 
 case "${ROLE}" in
   "master")
-    kubeadm init --token "${TOKEN}" --apiserver-bind-port 443 --skip-preflight-checks --apiserver-advertise-address "$(get_metadata "k8s-advertise-addresses")" --kubernetes-version $KUBERNETES_VERSION
+    kubeadm init --token "${TOKEN}" --pod-network-cidr 10.244.0.0/16 --apiserver-bind-port 443 --skip-preflight-checks --apiserver-advertise-address "$(get_metadata "k8s-advertise-addresses")" --kubernetes-version $KUBERNETES_VERSION
```

Comment:
Are you two working together on the overall goal? These pull requests look incompatible with each other.